src/mjolnir/CMakeLists.txt

```diff
@@ -72,6 +59,14 @@ set(sources
   util.cc
   validatetransit.cc)
 
+if (UNIX)
+  # Enables stricter compiler checks on a file-by-file basis
+  # which allows us to migrate piecemeal
+  set_source_files_properties(
+          ${sources}
+          PROPERTIES COMPILE_FLAGS "-Werror -Wall -Wextra -Wno-missing-field-initializers -Wno-unused-parameter")
```

Comment:
Were there too many warnings about unused parameters? Since you enable `Wno-unused-parameter`?

Reply:
@PureW I commented in the other thread to make the history smooth, check this out pls https://github.com/valhalla/valhalla/pull/2995#discussion_r637993339